cypress/integration/addtodo_bonus_spec.js

```diff
@@ -0,0 +1,39 @@
+describe("My First Test", () => {
```

Comment:
Can you change `My First Test` to a more descriptive name?

Reply:
Like this?

```suggestion
describe("My Second Test", () => {
```

Reply:
jk - will update today :)